documentation/asciidoc/topics/con_services.adoc

```diff
@@ -0,0 +1,34 @@
+[id='services-{context}']
+= Service Types
+Services are stateful applications, based on the {brandname} server image, that
+provide flexible and robust in-memory data storage.
+
+There are two service types {cacheservice} or {datagridservice}.
+
+== {cacheservice}
+
+Use {cacheservice} if you want {brandname} to provide a volatile, low-latency
+data store that requires minimal configuration.
+
+By default, {cacheservice} nodes:
+
+* Synchronously distribute data across the cluster to ensure consistency.
+* Maintain single copies of cache entries to reduce size.
+* Store cache entries off-heap and use eviction for JVM efficiency.
+* Ensure data consistency with a default partition handling configuration.
+
+[IMPORTANT]
+====
+Restarting {cacheservice} nodes removes all data in the cache. Restarts occur
```

Comment:
@oraNod looks like not any update on the **Infinispan** CR will occurs restarts.

Reply:
Thanks, Dmitry. This note was a bit of a last minute request from the support team and it was kind of an awkward fit. I've reworked it to include as part of the shutdown/restart section.